tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java

```diff
@@ -1879,8 +1879,8 @@ public void testListConsumers() throws Exception {
          array = (JsonArray) consumersAsJsonObject.get("data");
 
          Assert.assertEquals("number of consumers returned from query", 2, array.size());
-         Assert.assertEquals("check consumers address", addressName1.toString(), array.getJsonObject(0).getString("address"));
-         Assert.assertEquals("check consumers address", addressName1.toString(), array.getJsonObject(1).getString("address"));
+         Assert.assertEquals("check consumers address", addressName1.toString(), array.getJsonObject(0).getString("queueAddress"));
```

Comment:
should keep this simply address IMO.

Reply:
Just for my benefit so it links this with the other PR, https://github.com/apache/activemq-artemis/pull/1628